ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -8,34 +8,43 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 
 expiration_date=$(date -d '4 hours' --iso=minutes --utc)
 
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
 REGION="${LEASED_RESOURCE}"
 case "${REGION}" in
 us-east-1)
-   ZONE_1=us-east-1b
-   ZONE_2=us-east-1c;;
+    ZONES=("us-east-1b" "us-east-1c")
+    ;;
+*)
+    ZONES=("${REGION}a" "${REGION}b" "${REGION}c")
```

Comment:
```suggestion
    ZONES=("${REGION}a" "${REGION}b")
```

so that we keep things as is for now i.e. support at max 2 zones. We can later on bump it for regions we know to have 3 zones..